src/ResourceManager/Resources/Commands.Resources/help/Get-AzureRmManagementGroup.md

```diff
@@ -0,0 +1,235 @@
+---
+external help file: Microsoft.Azure.Commands.ManagementGroups.dll-Help.xml
+Module Name: AzureRM.ManagementGroups
+online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.managementgroups/
```

Comment:
This actually needs to have the cmdlet name at the end: https://docs.microsoft.com/en-us/powershell/module/azurerm.managementgroups/get-azurermmanagementgroup.  Sorry for the lack of clarity

Apply everywhere.